client/cypress/e2e/projects.cy.ts

```diff
@@ -163,46 +164,46 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight }) =>
           .trigger('pointermove', { pageX: x - 20 })
           .trigger('pointerup');
         cy.get('[data-test=AllocationItem__removeButton]').should('be.visible');
-        cy.get('[data-test=AllocationItem__removeButton]').click();
+        cy.get('[data-test=AllocationItem__removeButton]').click({ force: true });
```

Comment:
Why `force: true`? It usually means UI is broken, CY is forced to click on an element that is unclickable for whatever reason.

Reply:
Fixed